test/gtest/ucp/test_ucp_memheap.cc

```diff
@@ -160,6 +224,10 @@ void test_ucp_memheap::test_blocking_xfer(blocking_send_func_t send,
         zero_offset = 1;
     }
 
+    if (GetParam().thread_type != SINGLE_THREAD) {
```

Comment:
can we add function uct_test::get_num_threads() to check `return (GetParam().thread_type == SINGLE_THREAD) ? 1 : MT_TEST_NUM_THREADS`
to avoid code duplication